go/packages/gem.go

```diff
@@ -0,0 +1,98 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package packages
+
+import (
+	"os/exec"
+	"runtime"
+	"strings"
+)
+
+var (
+	gem string
+
+	gemListArgs     = []string{"list", "--local"}
+	gemOutdatedArgs = []string{"outdated", "--local"}
+)
+
+func init() {
+	if runtime.GOOS != "windows" {
```

Comment:
for my knowledge, why do we need this check?

Reply:
Mainly for future proofing, and to be very clear these package managers are not setup for Windows. We may support gem and pip in the future though.

Functionally there is no difference if I left this check out as nothing currently evaluates this, and even if it did these paths would not exist on Windows.